google_compute_engine/instance_setup/instance_config.py

```diff
@@ -89,6 +89,7 @@ class InstanceConfig(config_manager.ConfigManager):
           'run_dir': '',
           'startup': 'true',
           'shutdown': 'true',
+          'default_shell': '/bin/bash',
```

Comment:
Please also update the `README` if you are introducing a new variable to the instance defaults file.

Reply:
Done